docs/en/sql-reference/sql-statements/data-definition/ALTER_TABLE.md

```diff
@@ -963,6 +1004,112 @@ The `be_compactions` table in the `information_schema` database records compacti
     MODIFY COLUMN v2 DOUBLE DEFAULT "1" AFTER v1;
     ```
 
+14. Add and drop fields in STRUCT-type data.
+
+    Create a table and insert a row of data.
+
+    ```sql
+    CREATE TABLE struct_test(
+        c0 INT,
+        c1 STRUCT<v1 INT, v2 STRUCT<v4 INT, v5 INT>, v3 INT>,
+        c2 STRUCT<v1 INT, v2 ARRAY<STRUCT<v3 INT, v4 STRUCT<v5 INT, v6 INT>>>>
+    )
+    DUPLICATE KEY(c0)
+    DISTRIBUTED BY HASH(`c0`) BUCKETS 1
+    PROPERTIES (
+        "fast_schema_evolution" = "true"
+    );
+    INSERT INTO struct_test VALUES (
+        1, 
+        ROW(1, ROW(2, 3), 4), 
+        ROW(5, [ROW(6, ROW(7, 8)), ROW(9, ROW(10, 11))])
+    );
+    ```
```

Comment:
Show a `SELECT *` to show the data, so users can do a comparation with the changed data below.